packages/connector-thegraph/src/core/GraphQLWrapper.ts

```diff
@@ -0,0 +1,44 @@
+import 'isomorphic-unfetch';
```

Comment:
This wrapper makes lots of sense regardless of the implementation we decide in my opinion.